pulp-core/requirements.txt

```diff
@@ -70,7 +70,7 @@ python-dateutil==2.8.1
 python-debian==0.1.39
 python-gnupg==0.4.7
 pytz==2021.1
-PyYAML==5.4.1
```

Comment:
why the downgrade?

Reply:
Need to update pulp-core to 3.11 to support the new PyYAML.  The "upgrade" was done by dependabot, and I forgot to revert it even though that breaks the build. :)